addons/mail/static/src/messaging/component/message/message_tests.js

```diff
@@ -128,33 +128,36 @@ QUnit.test('delete attachment linked to message', async function (assert) {
 
     await this.start();
     const message = this.env.entities.Message.create({
-        attachment_ids: [{
+        attachments: [['insert-and-replace', {
```

Comment:
'insert' was introduced as 'insert-and-link' in mind. the '-and-link' part has been omitted just because it made sense this was implicit for most commands (minus 'unlink' ofc). Then when I needed '-and-replace', that's when I had needed to be more explicit. I think all this make sense.

Reply:
This is exactly the problem I have actually. `insert` is always to be understood as `insert-and-link`, so here it looks like `insert-and-link-and-replace` which is confusing.

Not even mentioning that `insert` also has to be understood as `create-or-update` at data level and `replace` as `remove-all-and-link` at relation level.

I think there is just too much going on at once here.